src/alire/os_linux/alire-platforms-current__linux.adb

```diff
@@ -20,7 +22,22 @@ package body Alire.Platforms.Current is
    begin
       if Distro_Cached then
          return Cached_Distro;
-      elsif not GNAT.OS_Lib.Is_Regular_File (OS_Identity_File) then
+      end if;
+
+      --  Try to get the distribution ID from an environment variable
+      --  Rationale: Map AlmaLinux, CentOS, Rocky Linux, Oracle Linux
+      --  et al to RHEL.
+
+      begin
+         Cached_Distro := Alire.Platforms.Distributions'Value
+           (Ada.Environment_Variables.Value ("ALIRE_DISTRIBUTION"));
```

Comment:
This `ALIRE_DISTRIBUTION` should be defined in `Alire.Environment`

Reply:
@mosteo Do we have a naming convention for variables that are inputs to the `alr` command vs variables that are set/defined by the `alr` command? I think it will get confusing otherwise.

Reply:
And we are also setting `ALIRE=True`